DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -35,17 +31,13 @@ List<DelegateUser> delegateUsers
         );
 
         void UpdateUserDetailsAndCentreSpecificDetails(
-            int userId,
             MyAccountDetailsData myAccountDetailsData,
-            CentreAnswersData centreAnswersData
+            DelegateDetailsData? delegateDetailsData,
+            string? centreEmail,
+            int? centreId

```

Comment:
I think it makes sense to put centreId in DelegateDetailsData - it is in the DelegateAccounts table after all.

Reply:
Users can have a centre email without a delegate account, we need it out